filebeat/module/elasticsearch/audit/test/test-audit-730.log-expected.json

```diff
@@ -18,7 +18,7 @@
         "event.action": "access_granted",
         "event.dataset": "elasticsearch.audit",
         "event.module": "elasticsearch",
-        "event.timezone": "+00:00",
+        "event.timezone": "-02:00",
```

Comment:
I'm not sure this is correct. I think this should remain `+00:00`.

If you look at the first sample log entry in `test-audit-730.log`, it has the following field:

```
"timestamp":"2019-06-11T05:21:08,484-0700"
```

The corresponding field in `test-audit-730.log-expected.json` is:

```
"@timestamp": "2019-06-11T12:21:08.484Z"
```

So wouldn't we want `event.timezone` here to be `+00:00`?